pom.xml

```diff
@@ -58,6 +58,10 @@
       <artifactId>snappy-java</artifactId>
       <version>${snappy-java.version}</version>
     </dependency>
+    <dependency>
```

Comment:
Ok yes I guess for the API call to the annotation processing service we use a different client (openfeign I believe). But that will be removed at some point and then this will be the only client.